builtins.c

```diff
@@ -288,12 +288,14 @@ static value_t fl_vector_alloc(value_t *args, u_int32_t nargs)
         f = args[1];
     else
         f = FL_UNSPECIFIED;
+    fl_gc_handle(&f);
     v = alloc_vector((unsigned)i, f==FL_UNSPECIFIED);
     if (f != FL_UNSPECIFIED) {
```

Comment:
I think it would be better to rewrite this as

```
v = alloc_vector((unsigned)i, 0);
if (nargs == 2)
    f = args[1];
else
    f = FL_UNSPECIFIED;
int k;
for(k=0; k < i; k++)
    vector_elt(v,k) = f;
```

A gc handle should not be necessary, since value is already rooted on the stack.